cmd/expired-authz-purger/main_test.go

```diff
@@ -33,7 +33,7 @@ func TestPurgeAuthzs(t *testing.T) {
 
 	p := expiredAuthzPurger{log, fc, dbMap, 1}
```

Comment:
(Adding a comment here vs at the top level of the review to make it more visible, unrelated to line 34 specifically)

Can you add something to the unit test (or add a new unit test) that exercises the ticker codepath? I think its undertested presently, all invocations use `maxDPS = 0`.